lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1,336 +1,194 @@
 // lms - views - shopping cart
 // ====================
 
+$enrollment-details-width-difference: $baseline*11.85;
```

Comment:
Ah ok, I understand now. In general we've used baseline more for padding / margin rules, whereas 237px sounds fairly specific to me and set by something else? I'd have to take a look on the sandbox to double check, but this is fine to leave as is for now. Thanks! 

Is there a course with payment set up on the sandbox I can review?